### PR TITLE
Updated Sector File Descriptor for new URL

### LINKED ISSUE #261: Change auto-updater file to use vatsim.uk

### PR DESCRIPTION
**Pull request by hsugden**
Fixes #261

# Summary of changes

With the changes to the UK domain, we now need to use vatsim.uk

This change fixes that

### CODE REVIEW

**Review by GeekPro101 — APPROVED** [review]

**Review by KHardern — CHANGES_REQUESTED** [review]
the shortened URL is just a redirect, which is likely to cause errors. Update the domain is fine.

have you tested this?

File: UK/Data/Sector/VATUK_Euroscope_files.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 EuroScope Sector File Provider File
 NAME:VATSIM-UK Euroscope Sector File Provider
-URL:http://www.vatsim-uk.co.uk/files/sector/esad/VATUK_Euroscope_files.txt
+URL:http://www.vatsim.uk/VATSIM_UK_Sector_Provider.txt
```
Comment (cpawley) [inline UK/Data/Sector/VATUK_Euroscope_files.txt:3]:
```suggestion
URL:http://www.vatsim.uk/files/sector/esad/VATUK_Euroscope_files.txt
```

File: UK/Data/Sector/VATUK_SectorFileProviderDescriptor.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 NAME:VATSIM-UK Euroscope Sector File Provider
-URL:http://www.vatsim-uk.co.uk/files/sector/esad/VATUK_Euroscope_files.txt
+URL:http://www.vatsim.uk/VATSIM_UK_Sector_Provider.txt
```
Comment (cpawley) [inline UK/Data/Sector/VATUK_SectorFileProviderDescriptor.txt:2]:
```suggestion
URL:http://www.vatsim.uk/files/sector/esad/VATUK_Euroscope_files.txt
```

**Comment by KHardern** [comment]
I'm going to commit what I think works in my other branch. I think the redirects work, but it needs an update of the download time to work properly.

**Review by luke11brown — APPROVED** [review]

**Pull request merged**